cmd/promcord/service.go

```diff
@@ -109,7 +122,9 @@ func main() {
 		log.From(ctx).Fatal("creating discord client", zap.Error(err))
 	}
 
-	discord.AddHandler(handler(ctx))
+	discord.AddHandler(messageCreateHandler(ctx))
```

Comment:
nit: the Handler part in messageCreateHandler is redundant, we could fix the naming and make it simpler